app/tray/Account/Requests/TransactionRequest/ViewData/index.js

```diff
@@ -36,7 +36,7 @@ const SimpleTxJSON = ({ json, req }) => {
           <div key={key + o} className='simpleJsonChild'>
             <div className=' simpleJsonKey simpleJsonKeyTx'>{key.replace(/([A-Z])/g, ' $1').trim()}</div>
             <div className='simpleJsonValue'>
-              {typeof json[key] === 'object' ? <SimpleJSON json={json[key]} key={key} /> : json[key]}
+              {json[key]}
```

Comment:
Is there no longer a case where this can be an object?

Reply:
I think this was to handle typed data which has sub-objects, but that uses different code now. mostly removed it because I noticed the import isnt even in the file so I don't think its used